Makefile

```diff
@@ -29,16 +29,6 @@ test-native:
 clean:
 	$(MAKE) mvn-clean
 
-.PHONY: zip-templates
-zip-templates:
-	(cd src/main/resources/static && zip -r templates.zip templates/)
```

Comment:
Can we also remove the folder `src/main/resources/static/templates` from the repository? This would reduce the size of the native executable.

Reply:
Yes, I figured I'd do the cleanup in a follow up PR, along with removing the scaffolding package as well.